requirements/edx/base.txt

```diff
@@ -77,6 +77,7 @@ django-pyfs==2.0
 django-ratelimit-backend==1.1.1
 django-ratelimit==2.0.0
 django-require==1.0.11
+django-rest-framework==0.1.0  # via super-csv
```

Comment:
This is not the correct DRF package.  Not sure what's going on but we should not merge this.

Reply:
Looks like it's due to a mistake here: https://github.com/edx/super-csv/pull/6/files